tox.ini

```diff
@@ -18,12 +15,13 @@ deps =
 
     checkqa: flake8
 
-    djangomaster: https://github.com/django/django/archive/master.tar.gz
-    django1.11: Django>=1.11,<1.12
-    django1.10: Django>=1.10,<1.11
-    django1.9: Django>=1.9,<1.10
-    django1.8: Django>=1.8,<1.9
-    django1.7: Django>=1.7,<1.8
+    djmaster: https://github.com/django/django/archive/master.tar.gz
+    dj20: Django>=2.0a1,<2.1
+    dj111: Django>=1.11,<1.12
+    dj110: Django>=1.10,<1.11
+    dj19: Django>=1.9,<1.10
+    dj18: Django>=1.8,<1.9
+    dj17: Django>=1.7,<1.8
```

Comment:
django 1.7 should be removed

Reply:
I opened a separate PR for that: pytest-dev/pytest-django#524